tool/tbot/main.go

```diff
@@ -158,6 +166,18 @@ func Run(args []string, stdout io.Writer) error {
 		"Arguments to `tsh proxy ...`; prefix with `-- ` to ensure flags are passed correctly.",
 	))
 
+	sshCmd := app.Command("ssh", "").Hidden()
+	sshCmd.Flag("destination-dir", "The destination directory with which to authenticate tsh").StringVar(&cf.DestinationDir)
+
+	sshProxyCmd := sshCmd.Command("proxy", "")
+	sshProxyCmd.Arg("[user@]host:port", "Remote host:port and the login to use").Required().StringVar(&cf.UserHostPort)
+	sshProxyCmd.Flag("cluster", "The cluster name. Extracted from the certificate if unset.").StringVar(&cf.Cluster)
+	sshProxyCmd.Flag("proxy-server", "The Teleport proxy server to use, in host:port form.").Required().StringVar(&cf.ProxyServer)
+	sshProxyCmd.Flag("tls-routing", "Whether the Teleport cluster has tls routing enabled.").Required().BoolVar(&cf.TLSRoutingEnabled)
+	sshProxyCmd.Flag("connection-upgrade", "Whether the Teleport cluster requires an ALPN connection upgrade.").Required().BoolVar(&cf.ConnectionUpgradeRequired)
+	sshProxyCmd.Flag("tsh-config", "The path to a tsh config file.").Required().StringVar(&cf.TSHConfigPath)
```

Comment:
Can we make this more explicit that it is meant to be a path to a proxy templates file - rather than specifically a tsh-config ?

Reply:
Yeah - but I think down the line I may adjust this so it doesn't support the whole tsh config - i'm happy for us to re-use that now for the aim of speed but I think that if we set that as the parameter name, people may start expecting other config options with the tsh config to work.

Reply:
I'd worry about --config-file being conflated with the standard config for tbot or being conflated with the soon-to-exist output metadata file. I think having this be `--proxy-templates` means that once the output metadata file is introduced, there's a sane override behaviour where any proxy templates inside the output metadata will be ignored and the proxy templates file provided at `--proxy-templates` will be used instead.